python/lsst/ci/hsc/gen2/validate.py

```diff
@@ -409,7 +421,17 @@ def validateSources(self, dataId):
         # that flag propagation, aperture correction, and extendendess are all
         # running and configured reasonably (but it may not be sensitive
         # enough to detect subtle bugs).
-        self.checkPsfStarsAndFlags(catalog, minStellarFraction=0.90, doCheckFlags=False)
+        # 2020-1-13: There is an issue with the PSF that was
+        # identified in DM-28294 and will be fixed in DM-12058,
+        # which affects scarlet i-band models. So we set the
+        # minStellarFraction based on the deblender and band used.
+        # Once DM-12058 is merged this band-aid can be removed.
```

Comment:
You can put a TODO here, and you can put a comment on DM-12058 so that this band-aid can be removed as part of that ticket.